### PR TITLE
update ip@2.0.0 to ip@2.0.1

### PR DESCRIPTION
**Pull request by JedML**
**Description**

- updated ip@2.0.0 to ip@2.0.1

**Motivation**

CE has a CVE for ip2.0.0 we are updating ip@2.0.0 to ip@2.0.1

**Testing Done**

- `python3 build-locally.py linux64`

**Backwards Compatibility Criteria (if any)**

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by JedML** [comment]
@conda-forge-admin, please rerender

File: .azure-pipelines/azure-pipelines-linux.yml [hunk 1]
```diff
@@ -13,36 +13,9 @@ jobs:
...
-         sudo apt-get purge -y -f firefox \
-                                  google-chrome-stable \
-                                  microsoft-edge-stable
-         sudo apt-get autoremove -y >& /dev/null
-         sudo apt-get autoclean -y >& /dev/null
-         df -h
-    displayName: Manage disk space
-  # This creates swap file that creates temporary memory space when the system runs low on memory
-  - script: |
-      sudo fallocate -l 10GiB /swapfile || true
```
Comment (aws-prayags) [inline .azure-pipelines/azure-pipelines-linux.yml:41]: re rendering is also removing the workaround that was added for low memory

Comment (JedML) [reply .azure-pipelines/azure-pipelines-linux.yml:41]: adding back in

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
 package:
   name: {{ name|lower }}
   version: {{ version|replace("-", "_") }}
 
 source:
   # github repo set-up is in progress, once ready we will use the github release url here instead of the s3 url
-  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor.tar.gz
-  sha256: 9effda827c45e1fd8dfc534ce310791ee46070d818ede89abf19e3ec9c8906d0
+  url: https://sagemaker-code-editor-1696292125.s3.us-west-2.amazonaws.com/code-editor1.0.1.tar.gz
```
Comment (aws-spenceng) [inline recipe/meta.yaml:10]: nit: should be `code-editor-1.0.1`

File: .azure-pipelines/azure-pipelines-linux.yml [hunk 3]
```diff
@@ -13,6 +13,7 @@ jobs:
         UPLOAD_PACKAGES: 'True'
         DOCKER_IMAGE: quay.io/condaforge/linux-anvil-cos7-x86_64
   timeoutInMinutes: 360
+  variables: {}
```
Comment (aws-spenceng) [inline .azure-pipelines/azure-pipelines-linux.yml:16]: What is the `variables` `{}` for?

**Review by aws-spenceng — APPROVED** [review]

File: .ci_support/linux_64_.yaml [hunk 4]
```diff
@@ -23,3 +27,5 @@ target_platform:
 zip_keys:
 - - c_compiler_version
   - cxx_compiler_version
+- - c_stdlib_version
+  - cdt_name
```
Comment (arkaprava08) [inline .ci_support/linux_64_.yaml:31]: Why are we tinkering with these files?

File: .azure-pipelines/azure-pipelines-linux.yml (hunk 3 above)
Comment (JedML) [reply .azure-pipelines/azure-pipelines-linux.yml:16]: that was part of the re-render, not sure

File: .ci_support/linux_64_.yaml (hunk 4 above)
Comment (JedML) [reply .ci_support/linux_64_.yaml:31]: re-render updaates

**Review by aws-pangestu — APPROVED** [review]

**Review by arkaprava08 — APPROVED** [review]
LGTM

**Review by aws-pangestu — APPROVED** [review]

**Pull request merged**